ecodev_core/permissions.py

```diff
@@ -13,3 +13,5 @@ class Permission(str, Enum):
     ADMIN = 'Admin'
     Consultant = 'Consultant'
     Client = 'Client'
+    ClientAdmin = 'ClientAdmin'
```

Comment:
Can you add 
    FORM_ADMIN = 'Form Admin'
    VALIDATOR = 'Validator'
    APPRAISER = 'Appraiser'
    USER = 'User'
please